class/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age = 18, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] We can implement this `@id` so that it is auto-generated. Kindly remove it from the parameters list.

  ```
  def initialize(age, name = 'Unknown', parent_permission: true)
    @id = rand(1...1000)
  ```